Library-SMTLIB/src/de/uni_freiburg/informatik/ultimate/logic/LoggingScript.java

```diff
@@ -110,32 +127,28 @@ public LoggingScript(Script script, String file, boolean autoFlush,
 		} else {
 			out = new FileOutputStream(file);
 		}
-		mPw = new PrintWriter(
-				new BufferedWriter(new OutputStreamWriter(out)), autoFlush);
+		mPw = new PrintWriter(new BufferedWriter(new OutputStreamWriter(out)), autoFlush);
 		mLetter = useCSE ? new FormulaLet() : null;
 	}
 
-	private final Term formatTerm(Term input) {
+	private final Term formatTerm(final Term input) {
 		return mLetter == null ? input : new FormulaLet().let(input);
 	}
 
 	@Override
-	public void setLogic(String logic)
-		throws UnsupportedOperationException, SMTLIBException {
+	public void setLogic(final String logic) throws UnsupportedOperationException, SMTLIBException {
 		mPw.println("(set-logic " + logic + ")");
 		mScript.setLogic(logic);
```

Comment:
I think it is cleaner to call `super.setLogic(logic)` here and remove all occurrences of mScript in this file.